### PR TITLE
[Cloud Sync] Merge remote-tracking branch 'origin/master' 2021-10-12

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Merge master into cloud branch for cloud release

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: store/sqlstore/upgrade.go [hunk 1]
```diff
@@ -1683,3 +1685,55 @@ func hasMissingMigrationsVersion600(sqlStore *SqlStore) bool {
...
 }
+
+func upgradeDatabaseToVersion610(sqlStore *SqlStore) {
+	if hasMissingMigrationsVersion610(sqlStore) {
+		sqlStore.AlterColumnTypeIfExists("Sessions", "Roles", "text", "varchar(256)")
+		sqlStore.AlterColumnTypeIfExists("ChannelMembers", "Roles", "text", "varchar(256)")
+		sqlStore.AlterColumnTypeIfExists("TeamMembers", "Roles", "text", "varchar(256)")
+	}
+
+	// if shouldPerformUpgrade(sqlStore, Version5380, Version600) {
```
Comment (streamer45) [inline store/sqlstore/upgrade.go:1696]: Just beware the versions here should be updated.

**Review by streamer45 — APPROVED** [review]

File: store/sqlstore/upgrade.go (hunk 1 above)
Comment (amyblais) [reply store/sqlstore/upgrade.go:1696]: @streamer45 @jwilander Would you be willing to make the update today as Ibrahim is off this week?

Comment (streamer45) [reply store/sqlstore/upgrade.go:1696]:
```suggestion
	// if shouldPerformUpgrade(sqlStore, Version600, Version610) {
```

**Comment by amyblais** [comment]
@jwilander Please help prioritize PR review for today.

**Review by jwilander — APPROVED** [review]

**Pull request merged**